storage-service/src/main/kotlin/com/icerockdev/service/storage/s3/IS3Storage.kt

```diff
@@ -49,6 +54,16 @@ interface IS3Storage {
     fun generateFileKey(): String {
         return UUID.randomUUID().toString().replace("-","/")
     }
+
+    fun getBucketPolicy(bucket: String): String?
+
+    fun putBucketPolicy(bucket: String, configure: PolicyBuilder.() -> Unit, confirmRemoveSelfBucketAccess: Boolean = false): Boolean
```

Comment:
transfer the function `configure`  as the last argument

Reply:
Done